packages/block-library/src/calendar/edit.js

```diff
@@ -19,10 +18,10 @@ const getYearMonth = memoize( ( date ) => {
 	if ( ! date ) {
 		return {};
 	}
-	const momentDate = moment( date );
+	const dateObj = new Date( date );
```

Comment:
Do we know what format(s) `date` is, or can be? Some date formats will work well cross-browser, while others... not so much 😅

If the date format is known, we should probably document it.

Reply:
That one will definitely work well everywhere, thanks! Looks great! 🙂

Reply:
Thank you for double-checking 🙌